### DISCUSSION TITLE
Long time no new release

### DISCUSSION THREAD

**Discussion by simohauml** [pywinauto/pywinauto#1241] [General]
It's been a long time since last release. Is there a plan for new release?

**Comment by vasily-v-ryabov** [comment]
Yes, 0.7.0 is planned with at least Linux `backend="atspi"` and some re-factoring. Some broken backward compatibility is pending their fixes from me. Also macOS backend could be included.

**Reply by vasily-v-ryabov** [reply]
There is no exact deadline, because I need to review `comtypes` changes and release them as well.

**Reply by simohauml** [reply]
Great work and thanks a lot! pywinauto is a great tool.

**Comment by manju1847** [comment]
Any updates on 0.7.0 version? Any tentative date for this new release?
